Apex_Unit_Testing/Apex_Language_Test Case/Apex1 Test Case/WorkingDaysTest.cls

```diff
@@ -0,0 +1,31 @@
+@isTest
+public class WorkingDaysTest 
+{
+    @isTest
+    static void testNumberOfWorkingDaysPositiveCase()
+    {
+        Date startDate = Date.newInstance(2020, 5, 1);
+        Date endDate = Date.newInstance(2020, 5, 10);
+        Integer workDays = WorkingDays.WorkingDays(startDate, endDate);
+        System.assertEquals(5, workDays);
+    }
+   
```

Comment:
if we checked the code coverage we easily Identified the remaining cases 
* start date on Saturday
* last day on Monday